components/_expanding-components/uncheck-radio-button.js

```diff
@@ -0,0 +1,37 @@
+export const UncheckRadioButton = function(elementClass) {
+  const expandingComponent = elementClass
+  let expandingElement
+
+  const cacheDom = function() {
+    expandingElement = document.querySelectorAll('.' + expandingComponent)
```

Comment:
expandingElement seems to be a collection not an element

Reply:
that is true, I will fix that